tm2unixtime.c

```diff
@@ -104,9 +104,10 @@ static void do_range_limit_days_relative(timelib_sll *base_y, timelib_sll *base_
 static int do_range_limit_days(timelib_sll *y, timelib_sll *m, timelib_sll *d)
 {
 	timelib_sll leapyear;
-	timelib_sll days_this_month;
 	timelib_sll last_month, last_year;
 	timelib_sll days_last_month;
+	int retval = 0;
+	int* days_per_month_current_year;
```

Comment:
Nit: I usually put the star with the variable name, like: `int *days_per_month_current_year`